test/e2e/tests/onboarding.spec.js

```diff
@@ -19,6 +28,188 @@ describe('MetaMask onboarding', function () {
       },
     ],
   };
+
+  it('Clicks create a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeCreateNewWalletOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
+      },
+    );
+  });
+
+  it('Clicks import a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeImportSRPOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
```

Comment:
same as the comment above, regarding adding an assertion

Reply:
Done